### DISCUSSION TITLE
Copy code button

### DISCUSSION THREAD

**Discussion by jakbin** [community/community#7733] [Mobile]
Please give copy code button for this , like browser .

![Screenshot_20211115-194317_GitHub.jpg](https://user-images.githubusercontent.com/65591973/141796848-e32329d1-3ae7-4f78-abf1-78aa2b19bc1b.jpg)

**Comment by brianlovin** [comment]
Thanks for the suggestion @jakbin — that could definitely be helpful. I'll pass this along to the team!

**Comment by alenuyee** [comment]
@tpope

**Comment by alenuyee** [comment]
>  #9666
